server-spi-private/src/main/java/org/keycloak/models/Constants.java

```diff
@@ -184,4 +184,7 @@ public final class Constants {
 
     //attribute name used to mark a client as realm client
     public static final String REALM_CLIENT = "realm_client";
+
+    //attribute name used to mark a temporary admin user/service account as temporary
+    public static final String TEMP_ADMIN_ATTR_NAME = "temporary_admin";
```

Comment:
```suggestion
    public static final String IS_TEMP_ADMIN_ATTR_NAME = "is_temporary_admin";
```

I'd rather see something like this as it emphasizes it's a boolean attribute.

Relevant even for the JS changes.

@Pepo48 @vmuzikar WDYT?

Reply:
`is_temporary_admin` works for me